### PR TITLE
FISH-752 Adds section on aggregated history and the added widget setting

### PR DESCRIPTION
**Pull request by jbee**
In connection with the changes in https://github.com/payara/monitoring-console/pull/30

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 1]
```diff
@@ -39,6 +39,43 @@ By default, this settings is enabled.
 NOTE: To remove only the monitoring data collection from the server without disabling the monitoring service itself delete the `monitoring-console-core.jar` from the `/glassfish/modules` folder of your installation(s) and restart the server.
 
 
+[[monitoring-console-history]]
+== Enabling Data Aggregation (History)
+By default InSight has a sliding window of data per metric of usually one minute length which contains 1 point per second.
+This allows typical "real-time" monitoring.
+This points represent the raw data as it is probed/collected.
```
Comment (eliotmartin) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:46]:
```suggestion
These points represent the raw data as it is probed/collected.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 2]
```diff
@@ -39,6 +39,43 @@ By default, this settings is enabled.
...
+In the most recent minute there are the raw (not aggregated) points per second.
+In the most recent hour there is 1 point per minute from aggregating the seconds within the last minute.
+In the most recent day there is 1 point per hour from aggregating the minutes within the last hour.
+In the most recent month there is 1 point per day from aggregating the hours within the last day.
+
+The aggregation of points must be enabled using the following asadmin command:
+
+		asadmin> set-monitoring-console-configuration --history-enabled=true
+
+This command starts the aggregation which starts to grow as time passes.
```
Comment (eliotmartin) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:61]:
```suggestion
This command starts the aggregation which increases over time.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 3]
```diff
@@ -39,6 +39,43 @@ By default, this settings is enabled.
...
+		asadmin> set-monitoring-console-configuration --history-enabled=true
+
+all aggregated data is discarded and cannot be restored.
+Equally history is lost when the server is restarted.
+
+The aggregated history feature is designed to keep memory usage fairly low.
+1000 metrics require in the range of single digit MBs of memory.
+So in most environments the extra memory and CPU effort for the history should not be an issue.
+
+In the future we might look into persisting the aggregated history for further features.
```
Comment (eliotmartin) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:76]: `In the future we might look into persisting the aggregated history for further features.`

I don't like the idea of setting up expectation.  Docs should describe existing functionality _only_, not allude to something that might appear in the future.

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 4]
```diff
@@ -39,6 +39,43 @@ By default, this settings is enabled.
...
+When aggregation of points is disabled again using the following asadmin command
+
+		asadmin> set-monitoring-console-configuration --history-enabled=true
+
+all aggregated data is discarded and cannot be restored.
+Equally history is lost when the server is restarted.
+
+The aggregated history feature is designed to keep memory usage fairly low.
+1000 metrics require in the range of single digit MBs of memory.
+So in most environments the extra memory and CPU effort for the history should not be an issue.
```
Comment (eliotmartin) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:74]:
```suggestion
In typical environments the extra memory and CPU effort for the history should not be an issue.
```

**Review by eliotmartin — CHANGES_REQUESTED** [review]
Couple of minor things to address @jbee

**Review by OndroMih — APPROVED** [review]
Formally looks good, I approve. Please resolve @eliotmartin's comments.

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 5]
```diff
@@ -39,6 +39,43 @@ By default, this settings is enabled.
...
+		asadmin> set-monitoring-console-configuration --history-enabled=true
+
+all aggregated data is discarded and cannot be restored.
+Equally history is lost when the server is restarted.
+
+The aggregated history feature is designed to keep memory usage fairly low.
+1000 metrics require in the range of single digit MBs of memory.
+In typical environments the extra memory and CPU effort for the history should not be an issue.
+
+In the future we might look into persisting the aggregated history for further features.
```
Comment (jbee) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:76]:
```suggestion
```

**Comment by jbee** [comment]
@OndroMih Changes applied.

**Pull request merged**